physionet-django/console/templates/console/training_type/index.html

```diff
@@ -0,0 +1,74 @@
+{% extends "console/base_console.html" %}
+
+{% load static %}
+
+{% block title %}Training Types{% endblock %}
+
+{% block content %}
+ <div class="card mb-3">
+  <div class="card-header">
+    Training Types <span class="badge badge-pill badge-info">{{ training_types|length }}</span>
+  </div>
+  <div class="card-body">
+    <div><button type="button" class="btn btn-sm btn-danger" data-toggle="modal" data-target="#op-training">Create/Update On Platform Training</button></div>
```

Comment:
I'm not keen on phrase "On Platform Training". It's awkward to read and write, and I'm not sure it's necessary. 

I think probably just "training courses" is fine. We can discuss if we need to create a clear distinction between courses hosted on PhysioNet and external courses.